### DISCUSSION TITLE
No Code , Low Code

### DISCUSSION THREAD

**Discussion by vesacons** [formio/formio.js#5546] [General]
Hello friends, I am new to formiojs.

I am trying to develop a no code low code infrastructure for a corporate company. Do you think this is the right infrastructure?

I want to ask a technical question.

Can I add previously pre defined datasources for selectboxes?  (For Examle , Users, City,  Or the parameter names I defined before

Regards

**Comment by lane-formio** [comment]
I believe what you might be asking for would be our "resource" functionality found here: https://help.form.io/userguide/resources
